packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [28.1.0]
+
+### Added
+
+- Added token detection on 7 more networks ([#4184](https://github.com/MetaMask/core/pull/4184))
```

Comment:
Some of these PRs have improved change entries in the PR description, e.g."Token detection is now supported on cronos, celo, gnosis, fantom, polygon_zkevm, moonbeam, and moonriver." here.

The format isn't exactly the same as here though. Maybe we can combine both:

```suggestion
- Addtoken detection on 7 more networks ([#4184](https://github.com/MetaMask/core/pull/4184))
  - New supported networks are: cronos, celo, gnosis, fantom, polygon_zkevm, moonbeam, and moonriver
```

Reply:
Refactored in [80b8087](https://github.com/MetaMask/core/pull/4234/commits/80b8087d67eec36cd59a5e99c59c4b7a1c3d902c)!